compose-dev.yml

```diff
@@ -1,7 +1,7 @@
 citizen:
    build: .
-   dockerfile: core/Dockerfile
-   command: npm run start-citizen-dev
+   dockerfile: Dockerfile
```

Comment:
Yeah, I fixed the path when I moved files around without thinking too much about it. Removed in coming commit.